lib/constants.go

```diff
@@ -1317,6 +1318,13 @@ const (
 	MinAccessGroupKeyNameCharacters = 1
 	MaxAccessGroupKeyNameCharacters = 32
 
+	// TODO: What are reasonable values for these? Or do we even need them at all?
+	// Min/Max MaxNonceExpirationBlockHeightOffset - Min/max value to which the MaxNonceExpirationBlockHeightOffset can be set.
+	MinMaxNonceExpirationBlockHeightOffset = 1
+	MaxMaxNonceExpirationBlockHeightOffset = 10000
+	// TODO: what's our initial value for this in the event that the global params entry isn't set?
```

Comment:
Could just implicitly use MaxMaxNonceExpirationBlockHeightOffset as the default value when the entry doesn't exist. The max value is generally the safest acceptable value.